Documentation for entur public transport sensor

hfurubotten opened the pull request:
**Description:**

Documentation for new sensor `entur_public_transport`. Real-time departure times from public transport in Norway. 

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#17286

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

hfurubotten committed:
Documentation for entur public transport sensor

frenck committed:
:pencil2: Tweaks

frenck commented:
Thanks, @hfurubotten! 👍 

✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

hfurubotten committed:
Updated for changes in sensor code.
- Show on map configuration
- Expand platforms under stops.
- Bumped version number.

frenck reviewed (dismissed):
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

hfurubotten committed:
renamed file and updated for changes in config on the sensor.

hfurubotten committed:
bumped version number and minor tweaks.

fabaff committed:
Update 'ha_release'

fabaff reviewed (approved):
Thanks :bird:
